### PR TITLE
Port the proxy to logrus

### PR DESCRIPTION
**Pull request by dlespiau**
One step along providing structured logging for CC, port the proxy to logrus. This PR doesn't try hard to be be fully structured but is quite mechanical with its port. Further work is needed to properly equip messages with the metadata we care about (for instance having the POD/VM id in most message to be able to filter with it).

This PR has a small conflict with PR #https://github.com/clearcontainers/proxy/pull/56 and will need to be rebased before merging.

### CODE REVIEW

File: proxy.go [hunk 1]
```diff
@@ -95,21 +96,23 @@ type client struct {
 	conn net.Conn
 }
 
-func (c *client) info(lvl glog.Level, msg string) {
-	if !glog.V(lvl) {
-		return
-	}
-	glog.Infof("[client #%d] %s", c.id, msg)
-}
+var nextClientID = uint64(0)
```
Comment (jodh-intel) [inline proxy.go]: Err - is this PR on top of #56?

Comment (dlespiau) [inline proxy.go]: not quite, I splitted #56 and this PR will need to be rebased on top (as noted in the PR message).

Comment (dlespiau) [inline proxy.go]: (and now rebased!)

**Comment by jodh-intel** [comment]
`Anything.To().get().Rid().Of(0).Those(1)` `glog.V(1).Info()` calls ;-)

lgtm.

**Comment by sboeuf** [comment]
LGTM

**Comment by amshinde** [comment]
lgtm

**Pull request merged**